package/yaft/package

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(yaft)
+pkgdesc="Framebuffer terminal emulator with an on-screen touch keyboard"
+url=https://github.com/timower/rM2-stuff/tree/master/apps/yaft
+pkgver=0.0.1-1
```

Comment:
I see that the upstream of your fork has a versioning scheme (currently at 0.2.8). Do you plan on following that same versioning scheme? Or having your own numbering?

Reply:
I won't keep in sync with upstream, my fork diverges too much. Maybe I'll start at 0.3.0, as upstream hasn't had an update in quite some time.